README.md

```diff
@@ -153,7 +153,75 @@ Example output:
 
 # Usage in CI Systems
 
-> 🚧 COMING SOON 🚧
+> 🚧 COMING SOON: native docker image & GitHub action 🚧
+
+## GitHub Actions workflows
+
+Here is an example of a simaple GitHub Actions workflow that uses `kubectl-validate` to validate Kubernetes manifests.
+This workflow will run on every pull request to the `main` branch and will fail if any of the manifests in the dir `k8s-manifest/` are invalid.
+
+```yaml
+name: kubectl-validate
+on:
+  pull_request:
+    branches:
+      - main
+  
+env:
+  MANIFESTS_PATH: 'k8s-manifest/*.yaml'
+  SCHEMA_VERSION: '1.23'
+  
+jobs:
+  k8sManifestsValidation:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo content
+        uses: actions/checkout@v3
+        
+      - name: Setup go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.20'
+          
+      - name: Install kubectl-validate
+        run: go install sigs.k8s.io/kubectl-validate@latest
+        
+      - name: Run kubectl-validate
+        run: kubectl-validate $MANIFESTS_PATH --version $SCHEMA_VERSION | grep OK
```

Comment:
I'm skeptical about this command. Does it correctly treat a mixture of errors and successes?

I think using `--output json` and making sure `"status": "Failure"` is not present is more robust.

Reply:
> I think using --output json and making sure "status": "Failure" is not present is more robust.

Wait, the binary doesn't use exit code to show failure vs success? That's a massive short-coming ...

Reply:
That command will hide the output from the logs which is really unfortunate.

Reply:
I don't like it also, but as @apelisse mentioned, the program doesn't return a non-zero exit code (#61), so I needed to use the `grep` workaround to fail the job for invalid manifests.

@alexzielenski I'm not sure parsing the output as JSON will be a better option because this will make the failure reason less readable for humans ;)

I tested this with various scenarios (errors and successes), and it's working fine. *I think* it will be better to leave it like that, and once #61 is fixed, the `grep` workaround will not be needed anymore.

@apelisse the command will [not hide the output](https://github.com/eyarz/kubectl-validate-playground/actions/runs/5462310843/jobs/9941495771#step:5:8):
![image](https://github.com/kubernetes-sigs/kubectl-validate/assets/19731161/1c28650f-015f-4fc0-91f4-5860a88eadad)

 let me know if you want me to leave like that or change it to JSON